source/Cosmos.System2_Plugs/System/Drawing/ColorImpl.cs

```diff
@@ -0,0 +1,184 @@
+                    break;
+                case "AliceOrange":
+                    return Color.DarkOrange;
+                    break;
+                case "DarkOrchid":
+                    return Color.DarkOrchid;
+                    break;
+                case "DarkRed":
+                    return Color.DarkRed;
+                    break;
+                case "DarkSalmon":
+                    return Color.DarkSalmon;
+                    break;
+                case "DarkSeaGreen":
+                    return Color.DarkSeaGreen;
+                    break;
+                case "DarkSlateBlue":
+                    return Color.DarkSlateBlue;
+                    break;
+                case "DarkSlateGray":
+                    return Color.DarkSlateGray;
+                    break;
+                case "DarkTurquoise":
+                    return Color.DarkTurquoise;
+                    break;
+                case "DarkViolet":
+                    return Color.DarkViolet;
+                    break;
+                case "DeepPink":
+                    return Color.DeepPink;
+                    break;
+                case "DeepSkyBlue":
+                    return Color.DeepSkyBlue;
+                    break;
+                case "DimGray":
+                    return Color.DimGray;
+                    break;
+                case "DodgerBlue":
+                    return Color.DodgerBlue;
+                    break;
+                case "Firebrick":
+                    return Color.Firebrick;
+                    break;
+                case "FloralWhite":
+                    return Color.FloralWhite;
+                    break;
+                case "ForestGreen":
+                    return Color.ForestGreen;
+                    break;
+                case "Fuchsia":
+                    return Color.Fuchsia;
+                    break;
+                case "Gainsboro":
+                    return Color.Gainsboro;
+                    break;
+                case "GhostWhite":
+                    return Color.GhostWhite;
+                    break;
+                case "Gold":
+                    return Color.Gold;
+                    break;
+                case "Goldenrod":
+                    return Color.Goldenrod;
+                    break;
+                case "Gray":
+                    return Color.Gray;
+                    break;
+                case "Green":
+                    return Color.Green;
+                    break;
+                case "GreenYellow":
+                    return Color.GreenYellow;
+                    break;
+                case "Honeydew":
+                    return Color.Honeydew;
+                    break;
+                case "HotPink":
+                    return Color.HotPink;
+                    break;
+                case "":
+                    throw new Exception("Color Name must be passed to FromName");
```

Comment:
You should change the type of the thrown exception to ArgumentException, so it would be easier to catch that specific exception.